docs/picongpu-docs-env.yml

```diff
@@ -0,0 +1,20 @@
+# Create a separate conda environment which fulfills the package requirements
+# for building the documentation
+# E.g. run `conda create --file picongpu-docs-env.yml`
+#
+
+name: picongpu-docs-env
+channels:
+    - conda-forge
+dependencies:
+    - sphinx_rtd_theme>=0.3.1
+    - recommonmark
+    - sphinx==2.0.1
+    - breathe>=4.12.0
+    - sphinxcontrib-programoutput
+    - sphinxcontrib-napoleon>=0.7
+    - pygments
+    - matplotlib
```

Comment:
why are those decencies below required?

Reply:
Since some plots in our documentation (see https://picongpu.readthedocs.io/en/0.4.3/models/field_ionization.html for instance) are created on-the-fly and are not saved as images.